server/controllers/users.js

```diff
@@ -71,7 +68,7 @@ export const exists = (req, res) => {
   } else {
     return models.User.findOne({ attributes: ['id'], where: { email } }).then(
       user => {
-        cache.set(email, Boolean(user));
+        cache.set(email, Boolean(user), tenMinutesInSeconds);
```

Comment:
I guess this is the one we should disable as it may cause issues with newly registered users. 

In all cases I don't really see the point of having this cache as `email` parameter will almost always be different.

Also this is a very light query and the (potential) gain of caching this could be overthrown by the fact it is easy to fill it up with trashy data and consume RAM.

Reply:
@Betree Yeah, let's do that.